packages/sui-bundler/README.md

```diff
@@ -74,8 +99,8 @@ You should create an entry file that assigns your lib to your desired namespace:
 
 ```js
 // umd.js
-import MyFancyModule from '/src/index.js'
-window.namespace.fancy = MyFancyModule
+import MyFancyModule from "/src/index.js";
```

Comment:
is there any special reason for this? I mean, it doesn't follow our style guide now.

it's not only here

Reply:
Automatically generated. I only edited the `readme.md` and run `npm run co`.
All the other files have been modified automatically.